pkg/vpcmodel/synthesisOutput.go

```diff
@@ -41,35 +45,47 @@ func (j *SynthesisOutputFormatter) WriteOutput(c1, c2 *VPCConfig,
 	return &SingleAnalysisOutput{Output: outStr, VPC1Name: c1.VPC.Name(), VPC2Name: v2Name, format: Synthesis, jsonStruct: all}, err
 }
 
-func handleNameAndType(resource EndpointElem, externals spec.SpecExternals) (
+func handleNameAndType(resource EndpointElem, externals spec.SpecExternals,
+	segments spec.SpecSegments, grouping bool) (
 	resourceName string,
 	resourceType spec.ResourceType) {
 	resourceName = resource.SynthesisResourceName()
+	resourceType = resource.SynthesisKind()
 	if resource.IsExternal() {
 		if structObj, ok := resource.(*groupedExternalNodes); ok {
 			// should be always true if src is external"
 			// later in aggregate we change the name with other vpc configs
 			externals[resourceName] = structObj.CidrOrAddress()
 		}
 	}
-	resourceType = resource.SynthesisKind()
+	if grouping && !resource.IsExternal() {
+		if structObj, ok := resource.(*groupedEndpointsElems); ok {
+			// should be always true if src is internal"
+			// later in aggregate we change the name with other vpc configs
+			if structObj.LenOfGroupedEndpoints() > 1 {
```

Comment:
```suggestion
			if len(*structObj) > 1 {
```
No need for a dedicated function